lib/bedrock/index.js

```diff
@@ -1,26 +1,13 @@
-const buildIndexFromArray = require('../indexer')
-
 module.exports = (data) => {
   return {
```

Comment:
I mean put the functions before this return statement

Reply:
Also if you make a change to the PR that needs review, feel free to comment so we can get notified of it

Reply:
Got it! Thank you, will note

Reply:
> Also if you make a change to the PR that needs review, feel free to comment so we can get notified of it

Might've done it